### *_Умова задачі_*

*Є можливість вибору виробництва та реалізації двох наборів товарів широкого вжитку. За даними відділу маркетингу, яким були проведені дослідження ринку, можливий прибуток від виробництва і реалізації Х і Y наведено в таблицях(Х,Y - прибуток у грощових одиницях).*
| X |1000 |1500|2000|
|---|-----|----|----|
| P |0.5  |0.3 |0.2 |

| Y |1000 |1500|1750|
|---|-----|----|----|
| P |0.4  |0.4 |0.2 |

*Потрібно оцінити ступінь ризику та запропонувати рішення щодо випуску та реалізації одного із наборів товарів.*

### *_Рішення_*

####  *_1. Розрахуємо математичне сподівання (очікуваний дохід) для обох груп товарів_*

In [3]:
M_x = 1000*0.5 + 1500*0.3 + 2000*0.2
M_x

1350.0

In [4]:
M_y = 1000*0.4 + 1500*0.4 + 1750*0.2
M_y

1350.0

####  *_2. Розрахуємо дисперсію (середньоквадратичне відхилення від очікуваного доходу) для обох груп товарів_*

In [5]:
D_x = (1000**2*0.5 + 1500**2*0.3 + 2000**2*0.2) - M_x**2
D_x

152500.0

In [6]:
D_y = (1000**2*0.4 + 1500**2*0.4 + 1750**2*0.2) - M_y**2
D_y

90000.0

####  *_3. Розрахуємо стандартне відхилення (ступінь ризику) для обох груп товарів_*

In [8]:
σ_x = D_x**(1/2)
σ_x


390.51248379533274

In [9]:
σ_y = D_y**(1/2)
σ_y

300.0

### *_Висновок_*

*Враховуючи те, що очікуваний дохід по обом групам товарів є однаковий (1350), але  ступінь ризику по другій групі товарів (300) менше ніж по першій групі товарів (390.51), другий варіант є менш ризикованим і рекомендований до випуску і реалізації.*